lib/socket_proxy/receiver.ex

```diff
@@ -4,7 +4,12 @@ defmodule SocketProxy.Receiver do
 
   @behaviour :ranch_protocol
 
-  defstruct socket: nil, destination_pids: [], port: nil
+  @staleness_check_interval_ms Application.compile_env(
```

Comment:
Ooh, `compile_env`. I was going to call out using the `Application` environment in a module attribute, but actually it looks like that's okay now, using this macro. Huh.

Reply:
I picked up the "`compile_env` in a module attribute" pattern from code that Paul wrote, so it must be good. 😅